.github/workflows/build_and_test.yaml

```diff
@@ -111,7 +111,13 @@ jobs:
       - name: Build pg_duckdb extension
         id: build
         working-directory: duckdb
-        run: ERROR_ON_WARNING=1 make -j8 install
+        run: |
+          if [ '${{ matrix.version }}' = REL_17_STABLE ]; then
+            ERROR_ON_WARNING=1 make -j8 install DUCKDB_BUILD=Static
```

Comment:
It seems error-prone to only build one version that way no?

Reply:
I'm not too worried about the static library build of DuckDB working for one PG version, but not for another. I mainly added this CI coverage to make sure we don't accidentally break the static build logic in our makefile completely. So I think running it for just one version is fine. I'll add a code comment explaining that though.